index.html

```diff
@@ -110,7 +111,6 @@
 
     // subscribe to Frame-Timing and User-Timing events
     observer.observe({eventTypes: ['render', 'composite', 'mark', 'measure']});
-    observer.disconnect();
 
     // processing logic called by application code
     function processObservedPerfEvents() {
```

Comment:
The example never calls `processObservedPerfEvents()`.. The idea was to provide a "defer processing" example, but FWIW, perhaps that's unnecessary as a first intro to the API. My proposal would be to merge contents of that function into the PO callback?